protocols/kad/src/protocol.rs

```diff
@@ -250,15 +251,55 @@ where
                     };
                     future::ready(proto_to_resp_msg(response))
                 }),
-        )
+        ))
     }
 }
 
 /// Sink of responses and stream of requests.
-pub type KadInStreamSink<S> = KadStreamSink<S, KadResponseMsg, KadRequestMsg>;
+pub struct KadInStreamSink<S>(KadStreamSink<S, KadResponseMsg, KadRequestMsg>);
```

Comment:
Why do we need this wrapper type?

Reply:
This is because `KadOutStreamSink` is now stored in `KademliaHandlerEvent::Available` which implements `Debug`, thus `KadOutStreamSink` should as well. `KadInStreamSink` was changed similarly just to be consistent.

Reply:
I'd like to remove that event variant in any case so this can be reverted then, correct?